reference/api/tasks.md

```diff
@@ -115,19 +120,25 @@ This value is always `null` for `dumpCreation` tasks.
 
 #### `taskCancelation`
 
+| Name                  | Description                                                                                                                                                                                          |
+| :-------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`matchedTasks`**    | The number of tasks that can be canceled based on the request. If the API key doesn’t have access to any of the indexes specified in the request, those tasks will not be included in `matchedTasks` |
+| **`canceledTasks`**   | The number of tasks successfully canceled. If the task fails, this will be `0`                                                                                                                       |
+| **`originalFilters`** | The filter used in the [`/tasks/cancel`](#cancel-tasks) request                                                                                                                                      |
+
 #### `taskDeletion`
 
 ### `error`
 
 **Type**: Object
 **Description**: Error details and context. Only present when a task has the `failed` [status](#status)
```

Comment:
```suggestion
**Description**: Error details and context. Contains the error definition, if this task has `failed`. [status](#status). Otherwise, set to `null`
```